src/style.css

```diff
@@ -0,0 +1,70 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  background-color: #eee;
+}
+
+.header {
+  background-color: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  
+  height: 60px;
+  margin-top: 20px;
+  margin-left: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  position: relative;
+}
+
+.link__logo {
+  position: absolute;
+  left: 50px;
+}
```

Comment:
there is no need to use position absolute here, simply adding paddings to header solve your problem